packages/translations/webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+
+const is_serve   = process.env.BUILD_MODE === 'serve';
+const is_release = process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'staging';
+
+module.exports = {
+    entry: {
+        'i18n'  : path.resolve(__dirname, 'src', 'i18next/index.js'),
+        localize: path.resolve(__dirname, 'src', 'components/index.js'),
+    },
+    output: {
+        path         : path.resolve(__dirname, 'lib'),
+        filename     : '[name].js',
+        libraryExport: 'default',
+        library      : ['deriv-translations', '[name]'],
+        libraryTarget: 'umd',
+    },
+    optimization: {
+        minimize: true,
+        // TODO enable splitChunks
+        // splitChunks: {
+        //     chunks: 'all'
+        // }
+    },
+    devServer: {
+        publicPath: '/dist/',
+    },
+    devtool: is_release ? 'source-map' : 'cheap-module-eval-source-map',
+    module : {
+        rules: [
+            (!is_serve ? {
+                enforce: 'pre',
+                test   : /\.(js|jsx)$/,
+                exclude: /node_modules|lib|shared\/utils/,
+                include: /src/,
+                loader : 'eslint-loader',
+                options: {
+                    fix: true,
+                },
+            } : {}),
+            {
+                test   : /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                loader : 'babel-loader',
+            },
+        ],
+    },
+    externals: {
+        'react'                     : 'react',
+        'react-dom'                 : 'react-dom',
+        'react-pose'                : 'react-pose',
+        'babel-polyfill'            : 'babel-polyfill',
+        'prop-types'                : 'prop-types',
```

Comment:
are `react-pose` and `react-dom` used anywhere in the translation package? think it's ok to remove them

Reply:
correct, thanks @kelly-binary